app/src/main/java/org/simlar/contactsprovider/ContactsProvider.java

```diff
@@ -305,6 +304,7 @@ private static String createFakePhotoString()
 					continue;
 				}
 
+				String photoUri = null;
```

Comment:
Since you do really like the ternary operator, you could would use it here, too. Besides, `Uri.EMPTY` instead of `null` might be more null-safe.